scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/goals/RecurrenceGoal.java

```diff
@@ -69,8 +77,10 @@ protected RecurrenceGoal fill(RecurrenceGoal goal) {
       super.fill(goal);
 
       if (every == null) {
-        throw new IllegalArgumentException(
-            "creating recurrence goal requires non-null \"every\" duration interval");
+        throw new IllegalArgumentException("creating recurrence goal requires non-null \"every\" duration interval");
+      }
+      if (every.min.isNegative() || every.max.isNegative()) {
+        throw new IllegalArgumentException("Duration passed to RecurrenceGoal as the goal's recurrence interval cannot be negative!");
```

Comment:
Long term it would be nice for this error to specify which or the bounds are negative.